bundles/org.openhab.binding.tr064/src/main/feature/feature.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.tr064-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-tr064" description="TR-064 Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<feature>openhab.tp-jaxb</feature>
```

Comment:
A lot of bindings violate that: avmfritz, boschindego, chromecast, denonmarantz, dlinksmarthome, groheondus, icalendar, ipcamera, lgwebos, tellstick, xmltv, homekit, jinja, pollytts with openhab.tp-jaxb, openhab.tp-netty, openhab.tp-jackson, openhab.tp-jaxws.

Wouldn't it make more sense to fix them all at once after we discussed where such features should live? When the jaxb and jaxws features were introduced with https://github.com/openhab/openhab-core/pull/920 the feature-names were changed from my original  proposal `openhab-jaxb` and `openhab-jaxws` (cf. https://github.com/openhab/openhab-core/pull/684) to `openhab.tp-jaxb` and `openhab.tp-jaxws`. What is different now?

Reply:
Ok, maybe you are right - we haven't really clearly defined anywhere how to deal with "common" dependencies that are used by multiple add-ons, so clearly the way for them is to use the tp-features right now.
Indeed, the features within the openhab-core Karaf feature are all containing some openhab-core bundles as well, what makes them somewhat different.

I'd like to ask you though to declare the dependency on tp-features in the same way as it is done from the openhab-core features - i.e. [declaring a requirement](https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-core/src/main/feature/feature.xml#L132) and only add the feature [with dependency="true"](https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-core/src/main/feature/feature.xml#L133), so that it is only installed, if the requirement isn't already satisfied in some other way.

```
		<requirement>openhab.tp;filter:="(feature= jaxb)"</requirement>
		<feature dependency="true">openhab.tp-jaxb</feature>
		<requirement>openhab.tp;filter:="(feature=jax-ws)"</requirement>
		<feature dependency="true">openhab.tp-jaxws</feature>
```

Reply:
Good point. Done.